doc/api/process.md

```diff
@@ -1426,13 +1426,26 @@ added: v0.1.16
 * {string}
 
 The `process.platform` property returns a string identifying the operating
-system platform on which the Node.js process is running. For instance
-`'darwin'`, `'freebsd'`, `'linux'`, `'sunos'` or `'win32'`
+system platform on which the Node.js process is running.
+
+Currently possible values are:
+
+* `'aix'`
+* `'darwin'`
+* `'freebsd'`
+* `'linux'`
+* `'openbsd'`
+* `'sunos'`
+* `'win32'`
 
 ```js
 console.log(`This platform is ${process.platform}`);
 ```
 
+*Note*: The value `'android'` may also be returned if the Node.js is built on
+the Android operating system. However, Android support in Node.js is considered
+to be experimental at this time.
```

Comment:
Should we link to the supported platforms table? (I forget where it is..)

Reply:
I cannot find that either...

Reply:
https://github.com/nodejs/node/blob/master/BUILDING.md#supported-platforms-1

Reply:
@gibfahn it seems that the table just list the supported platforms, but not the value of the API.